modules/contentpool_custom_elements/contentpool_custom_elements.libraries.yml

```diff
@@ -0,0 +1,31 @@
+main:
+  version: 1.0.0
+  css: {}
+  js:
+    'js/load-vue-components.js': {}
+  dependencies:
+    - contentpool_custom_elements/vue
+
+vue:
+  version: 2.5.16
+  js:
+    'https://cdn.jsdelivr.net/npm/vue@2.5.16/dist/vue.js':
```

Comment:
please add "vue" module for that and depend on its library. that's what we already do on the client side, see https://github.com/drunomics/contentpool-client/pull/15